api/api.go

```diff
@@ -239,7 +248,7 @@ func NewServerRoutes(s *Server) *http.ServeMux {
 			panic(err)
 		}
 
-		remh := NewRemoteHandlers(s.qriNode, receivers)
+		remh := NewRemoteHandlers(s.qriNode, s.cfg, receivers)
```

Comment:
I assume that in the future this will change to be just:

```
remh := NewRemoteHandlers(s.inst, receivers)
```

It's probably too big of a change to do all of that now, but how about a TODO that mentions this plan?